vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -76,6 +76,15 @@ using DynamicAutomationProperties = Microsoft::ReactNative::DynamicAutomationPro
 
 FrameworkElementViewManager::FrameworkElementViewManager(const Mso::React::IReactContext &context) : Super(context) {}
 
+XamlView FrameworkElementViewManager::CreateView(int64_t tag, const winrt::Microsoft::ReactNative::JSValueObject &props){
+  auto view = Super::CreateView(tag, props);
+  auto frameworkElem = view.try_as<xaml::FrameworkElement>();
+  onUnloadedRevoker = frameworkElem.Unloaded({this, &FrameworkElementViewManager::OnUnloaded});
```

Comment:
>FrameworkElementViewManager::OnUnloaded [](http://example.com/codeflow?start=53&length=39)

I wonder if this works correctly for virtual methods. Could you confirm that if the OnUnloaded is overriden in the derived class, then event will call the correct overload?

Ideally, it would be nice to add a test for it.

Reply:
If this does end up being what you do, whenever doing a `view.try_as<>` it's a good practice to do a null check on the result before using it. ie:

```
if (frameworkElem) {
  onUnloadedRevoker = frameworkElem.Unloaded({this, &FrameworkElementViewManager::OnUnloaded});
}